osu.Game/Overlays/Rankings/RankingsOverlayHeader.cs

```diff
@@ -0,0 +1,135 @@
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Allocation;
+
+namespace osu.Game.Overlays.Rankings
+{
+    public class RankingsOverlayHeader : TabControlOverlayHeader<RankingsScope>
+    {
+        public readonly Bindable<RulesetInfo> Ruleset = new Bindable<RulesetInfo>();
+        public readonly Bindable<Spotlight> Spotlight = new Bindable<Spotlight>();
+        public readonly Bindable<Country> Country = new Bindable<Country>();
+
+        public IEnumerable<Spotlight> Spotlights
+        {
+            get => spotlightsContainer.Spotlights;
+            set => spotlightsContainer.Spotlights = value;
+        }
+
+        protected override ScreenTitle CreateTitle() => new RankingsTitle
+        {
+            Scope = { BindTarget = Current }
+        };
+
+        protected override Drawable CreateTitleContent() => new OverlayRulesetSelector
+        {
+            Current = Ruleset
+        };
+
+        private SpotlightsContainer spotlightsContainer;
+
+        protected override Drawable CreateContent() => new FillFlowContainer
+        {
+            RelativeSizeAxes = Axes.X,
+            AutoSizeAxes = Axes.Y,
+            Direction = FillDirection.Vertical,
+            Children = new Drawable[]
+            {
+                new CountryFilter
+                {
+                    Current = Country
+                },
+                spotlightsContainer = new SpotlightsContainer
+                {
+                    Spotlight = { BindTarget = Spotlight }
+                }
+            }
+        };
+
+        protected override void LoadComplete()
+        {
+            Current.BindValueChanged(onCurrentChanged, true);
+            base.LoadComplete();
+        }
+
+        private void onCurrentChanged(ValueChangedEvent<RankingsScope> scope) =>
+            spotlightsContainer.FadeTo(scope.NewValue == RankingsScope.Spotlights ? 1 : 0, 200, Easing.OutQuint);
+
+        private class RankingsTitle : ScreenTitle
+        {
+            public readonly Bindable<RankingsScope> Scope = new Bindable<RankingsScope>();
+
+            public RankingsTitle()
+            {
+                Title = "ranking";
+            }
+
+            protected override void LoadComplete()
+            {
+                base.LoadComplete();
+                Scope.BindValueChanged(scope => Section = scope.NewValue.ToString().ToLowerInvariant(), true);
+            }
+
+            protected override Drawable CreateIcon() => new ScreenTitleTextureIcon(@"Icons/rankings");
+        }
+
+        private class SpotlightsContainer : CompositeDrawable
+        {
+            public readonly Bindable<Spotlight> Spotlight = new Bindable<Spotlight>();
+
+            public IEnumerable<Spotlight> Spotlights
+            {
+                get => dropdown.Items;
+                set => dropdown.Items = value;
+            }
+
+            private readonly OsuDropdown<Spotlight> dropdown;
```

Comment:
Is this a placeholder for #7488? I'm not getting something here.

Reply:
Yeah, it's just a placeholder for now. Didn't want to spend much time for it since it will be updated anyway

Reply:
Currently we have the same thing, so not a big deal, I suppose